### DISCUSSION TITLE
Tailwind CSS changed the way I develop

### DISCUSSION THREAD

**Discussion by samrobbins85** [tailwindlabs/tailwindcss#3210] [Kind words]
Before tailwind all my custom designs looked pretty terrible, but now I can rapidly develop designs that look pretty good. Such an amazing tool, thank you so much!

**Comment by simonswiss** [comment]
Super nice to hear that Tailwind CSS empowered you to build better looking designs ❤️

**Reply by samrobbins85** [reply]
And thanks for the videos you've done, they've been great for understanding some of the more complex topics 😍
